Readme.md

```diff
@@ -24,14 +24,20 @@ Checksum utility for node.
          console.log('yay yay')
        }
     })
+    
+    checksum.file('dshaw.txt', { algorithm: 'md5' }, function (err, sum) {
+       if (sum === 'a5ad66d8992ac4842fee8a231afe4fdb') {
+         console.log('yay yay')
+       }
+    })
 
 ## Checksum cli tool
 
     npm install -g checksum
 
 ### Options
 
-* `-a` `--algorithm`: default `sha1`
+* `-a` `--algorithm`: default `sha1` | `sha256` | `md5`
```

Comment:
This is showing that the default value is one of `sha1`, `sha256`, and `md5`. In reality the default is `sha1`. Maybe the following is better. Try rewording the line a bit.